src/data/customer/Driver.js

```diff
@@ -50,6 +54,7 @@ export const createDriverAsync = async (
     password,
     phone,
     productTypeId,
+    //TODO: where is the sendConfirmationEmail?
```

Comment:
for v3 copied, there's no such option for this

Reply:
Frontend also needs to change when this is updated.